Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -3494,13 +3494,19 @@ internal static async Task PutObject_Test9(IMinioClient minio)
         var startTime = DateTime.Now;
         var bucketName = GetRandomName(15);
         var objectName = GetRandomObjectName(10);
-        var contentType = "application/octet-stream";
+        const int objSize = 1 * MB;
+        const string contentType = "application/octet-stream";
         var percentage = 0;
         var totalBytesTransferred = 0L;
         var progress = new Progress<ProgressReport>(progressReport =>
         {
             percentage = progressReport.Percentage;
             totalBytesTransferred = progressReport.TotalBytesTransferred;
+            // Console.WriteLine(
```

Comment:
Do we need these lines for reference?

Reply:
I left those as an example on how to print the progress information.
We don't want to introduce random print outs in between functional test results.

@shtripat 
Please review again as I've changed the way the size of the put object info is retrieved when the file is uploaded as a single file in one shot.